packages/@lwc/engine/src/framework/__tests__/vm.spec.ts

```diff
@@ -8,6 +8,7 @@ import { compileTemplate } from 'test-utils';
 import { createElement, LightningElement } from '../main';
 import { ViewModelReflection } from '../utils';
 import { getComponentVM } from '../vm';
+import { registerDecorators } from '../decorators/register';
```

Comment:
We should be consistent and import `createElement`, `LightningElement`, and `registerDecorators` from the same path.